tests/e2e/app/main.py

```diff
@@ -32,7 +38,19 @@ def capture():
 
 
 class TestRunnerHandler(webapp2.RequestHandler):
+
     def get(self):
+        # Set up test directory.  Done once per app upload.
+        if not os.path.isdir('tests'):
+          os.mkdir('tests')
+
+        # Refresh the tests from cloud storage.
+        bucket = storage.Client().get_bucket(GCLOUD_STORAGE_BUCKET)
+        blob_iter = bucket.list_blobs()
+        for blob in blob_iter:
+          blob.download_to_filename('tests/%s' % blob.name)
```

Comment:
On upload, we use gsutil rm bucket/** to delete everything in cloud storage.
On download, we first clear the tests directory with shutil.rmtree which is equivalent to 'rm -rf'